micronota/commands/database.py

```diff
@@ -6,22 +6,46 @@
 # The full license is in the file COPYING.txt, distributed with this software.
 # ----------------------------------------------------------------------------
 
+import os
+import pkgutil
+import importlib
+
 import click
 
 from ..cli import cmd, AliasedGroup
+from .. import db
 
 
 @cmd.group(cls=AliasedGroup)
 @click.pass_context
 def cli(ctx):
     '''Database operations.'''
+    print(ctx.parent.params)
     pass
 
 
-@cli.command('update')
-@click.option('-i', '--input', type=click.STRING)
-@click.option('-o', '--output', type=click.STRING)
+@cli.command('prepare')
+@click.argument('databases', nargs=-1)
+@click.option('-f', '--force', is_flag=True,
+              help='Force overwrite.')
 @click.pass_context
-def update_db():
-    '''Update database.'''
-    pass
+def create_db(ctx, databases, force):
+    '''Prepare database.'''
```

Comment:
Note that the docstring of the function is used as the documentation for the CLI i.e when you call `./something --help`. So it may be worth adding any other pertinent information here.